src/Agent.Sdk/Blob/PipelineTelemetryRecord.cs

```diff
@@ -19,12 +18,12 @@ public abstract class PipelineTelemetryRecord : BlobStoreTelemetryRecord
         public Guid JobId { get; private set; }
         public Guid TaskInstanceId { get; private set; }
 
-        public PipelineTelemetryRecord(TelemetryInformationLevel level, Uri baseAddress, string eventNamePrefix, string eventNameSuffix, AgentTaskPluginExecutionContext context, uint attemptNumber = 1)
+        public PipelineTelemetryRecord(TelemetryInformationLevel level, Uri baseAddress, string eventNamePrefix, string eventNameSuffix, IKnobValueContext context, uint attemptNumber = 1)
```

Comment:
Did we need to change this file or just cleanup?

Reply:
This file was moved into the shared library